README.md

```diff
@@ -1 +1,41 @@
-# angular-replace-readme
\ No newline at end of file
+# Replacing base markup
+
+## Overview
+
+Another property we can use is `replace` - let's look into that!
```

Comment:
Add more context to the overview - briefly mention what the problem is and how replace can help.